kernel/kernel-api/src/main/java/io/delta/kernel/internal/ScanImpl.java

```diff
@@ -94,7 +94,7 @@ public ScanImpl(
      */
     @Override
     public CloseableIterator<FilteredColumnarBatch> getScanFiles(Engine engine) {
-        return getScanFiles(engine, false);
```

Comment:
Let's make reading the tags by default. Most cases the contents are empty, so shouldn't add extra overhead for reads.